### PR TITLE
feat: User failsafe modal

### PR DESCRIPTION
**Pull request by savudani8**

### CODE REVIEW

**Comment by savudani8** [comment]
Screenshot from 2021-02-25 17-31-19

**Comment by nud3l** [comment]
Thanks for adding this! There are two things that need updating:

1. When a refund request is made, the "Deposited" amount needs to be the amount that the user sent (so not subtracting the fees paid to the vault). In the example below, I have sent 2 BTC, that should be shown next to "Deposited" and the difference should be 1 BTC. The requested amount of 0.99... BTC being sent back by the vault is correct.

image

2. When no refund request is made but I sent too much BTC, we need to display the actually issued amount. In the example here, I requested 0.1 PolkaBTC and sent 0.2 BTC. My balance is updated correctly, but in the case where the requestedAmount != issuedAmount, we need to show the issued amount.

This needs to be updated in the status modal:

image

And also in the table (issue-requests.tsx):

image

File: src/pages/app/issue/modal/whoops-view.tsx [hunk 1]
```diff
@@ -1,10 +1,109 @@
...
 export default function WhoopsView(props: WhoopsViewProps): ReactElement {
-    return <div></div>;
+    const { t } = useTranslation();
+    const { prices } = useSelector((state: StoreType) => state.general);
+
+    return (
+        <div className="status-view">
+            <React.Fragment>
+                <div className="row">
+                    <div className="col mt-4 refund-title">{t("issue_page.refund_whoops")}</div>
```
Comment (nud3l) [inline src/pages/app/issue/modal/whoops-view.tsx:22]: Can you please re-use "status-title" as class here?

File: src/pages/app/issue/modal/whoops-view.tsx [hunk 2]
```diff
@@ -1,10 +1,109 @@
...
 
 export default function WhoopsView(props: WhoopsViewProps): ReactElement {
-    return <div></div>;
+    const { t } = useTranslation();
+    const { prices } = useSelector((state: StoreType) => state.general);
+
+    return (
+        <div className="status-view">
+            <React.Fragment>
+                <div className="row">
```
Comment (nud3l) [inline src/pages/app/issue/modal/whoops-view.tsx:21]: leave out this row

File: src/pages/app/app.page.scss [hunk 3]
```diff
@@ -222,10 +223,47 @@
     .wizzard-number {
         color: black;
         font-size: 3rem;
-        font-weight: 600;
+        font-weight: 400;
+        font-family: airbnb-cereal-medium;
     }
 }
 
+.refund-title {
```
Comment (nud3l) [inline src/pages/app/app.page.scss:231]: same as "status-title"?

File: src/pages/app/app.page.scss [hunk 4]
```diff
@@ -222,10 +223,47 @@
...
+}
+
+.refund-subtitle {
+    font-family: airbnb-cereal-bold;
+    text-align: center;
+    opacity: 0.5;
+    font-size: 1rem;
+}
+
+.refund-amount {
```
Comment (nud3l) [inline src/pages/app/app.page.scss:246]: same as "orange-amount"?

File: src/pages/app/app.page.scss [hunk 5]
```diff
@@ -222,10 +223,47 @@
...
+    font-family: airbnb-cereal-bold;
+    color: $d_orange;
+}
+
+.opaque {
+    opacity: 1;
+}
+
+.orange {
```
Comment (nud3l) [inline src/pages/app/app.page.scss:255]: "orange-text"?

File: src/pages/app/app.page.scss [hunk 6]
```diff
@@ -222,10 +223,47 @@
...
+    opacity: 0.5;
+    font-size: 1rem;
+}
+
+.refund-amount {
+    font-family: airbnb-cereal-bold;
+    color: $d_orange;
+}
+
+.opaque {
```
Comment (nud3l) [inline src/pages/app/app.page.scss:251]: "total-added-value"

File: src/pages/app/app.page.scss [hunk 7]
```diff
@@ -222,10 +223,47 @@
...
+
+.orange {
+    color: $d_orange;
+}
+
+.refund-step > * {
+    float: right;
+}
+
+.green {
```
Comment (nud3l) [inline src/pages/app/app.page.scss:263]: "green-text"

**Review by nud3l — APPROVED** [review]

**Review by nud3l — APPROVED** [review]

**Pull request merged**